src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -76,7 +80,7 @@ const InfiniteScroll = ({
       setPageArea(nextPageArea);
       setMultiColumn(nextMultiColumn);
     }
-  }, [pageHeight, step, show]);
+  }, [beginPage, endPage, items, step, show]);
```

Comment:
The intent is to recalculate pageHeight whenever begin and end pages are incremented/decremented. In this case they are triggers, rather than dependencies.

re: items - @IanKBovard, I can't recall, was items added here also as a trigger for when onMore collects additional items?

Reply:
that is correct. items is needed to recalculate pageheight.